### ISSUE TITLE
关于partial reid代码的问题

### ISSUE THREAD

**Issue by Mrkkew** [JDAI-CV/fast-reid#104]
你好，我在根据代码的指示运行partial reid的代码时，运行到下面的部分就卡住了。训练数据集和预训练模型都在，为什么会出现这种现象呢？

      )
      (2): Bottleneck(
        (conv1): Conv2d(2048, 512, kernel_size=(1, 1), stride=(1, 1), bias=False)
        (bn1): BatchNorm(512, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
        (conv2): Conv2d(512, 512, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1), bias=False)
        (bn2): BatchNorm(512, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
        (conv3): Conv2d(512, 2048, kernel_size=(1, 1), stride=(1, 1), bias=False)
        (bn3): BatchNorm(2048, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
        (relu): ReLU(inplace)
        (se): Identity()
      )
    )
  )
  (heads): DSRHead(
    (pool_layer): AdaptiveAvgPool2d(output_size=1)
    (occ_unit): OcclusionUnit(
      (MaxPool1): MaxPool2d(kernel_size=2, stride=2, padding=0, dilation=1, ceil_mode=False)
      (MaxPool2): MaxPool2d(kernel_size=4, stride=2, padding=0, dilation=1, ceil_mode=False)
      (MaxPool3): MaxPool2d(kernel_size=6, stride=2, padding=0, dilation=1, ceil_mode=False)
      (MaxPool4): MaxPool2d(kernel_size=8, stride=2, padding=0, dilation=1, ceil_mode=False)
      (mask_layer): Linear(in_features=2048, out_features=1, bias=False)
    )
    (MaxPool1): MaxPool2d(kernel_size=2, stride=2, padding=0, dilation=1, ceil_mode=False)
    (MaxPool2): MaxPool2d(kernel_size=4, stride=2, padding=0, dilation=1, ceil_mode=False)
    (MaxPool3): MaxPool2d(kernel_size=6, stride=2, padding=0, dilation=1, ceil_mode=False)
    (MaxPool4): MaxPool2d(kernel_size=8, stride=2, padding=0, dilation=1, ceil_mode=False)
    (bnneck): BatchNorm(2048, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
    (bnneck_occ): BatchNorm(2048, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)
    (classifier): Linear(in_features=2048, out_features=751, bias=False)
    (classifier_occ): Linear(in_features=2048, out_features=751, bias=False)
  )
)
[06/14 01:59:01 fastreid.engine.defaults]: Prepare training set
[06/14 01:59:01 fastreid.data.datasets.bases]: => Loaded Market1501
[06/14 01:59:01 fastreid.data.datasets.bases]:   ----------------------------------------
[06/14 01:59:01 fastreid.data.datasets.bases]:   subset   | # ids | # images | # cameras
[06/14 01:59:01 fastreid.data.datasets.bases]:   ----------------------------------------
[06/14 01:59:01 fastreid.data.datasets.bases]:   train    |     0 |        0 |         0
[06/14 01:59:01 fastreid.data.datasets.bases]:   ----------------------------------------

**Comment by Mrkkew** [comment]
是因为找不到训练数据集吗？还有，在训练完之后的测试命令是什么样的呢？

**Comment by L1aoXingyu** [comment]
应该是数据集没有按照正确的格式存放，所以数据统计那里是空的

测试的话，直接
```
CUDA_VISIBLE_DEVICES=gpus python train_net.py --config-file <configs.yaml> --eval-only MODEL.WEIGHTS model.pth
```

**Comment by Hanswufe** [comment]
你好，请问你是怎么安装fastreid的，我看readme那块是..

**Comment by L1aoXingyu** [comment]
不需要安装 fastreid，fastreid 就是这个库

**Comment by Mrkkew** [comment]
> 应该是数据集没有按照正确的格式存放，所以数据统计那里是空的
> 
> 测试的话，直接
> 
> ```
> CUDA_VISIBLE_DEVICES=gpus python projects/PartialReID/train_net.py --config-file <configs.yaml> --eval-only MODEL.WEIGHTS model.pth
> ```
你好，这个问题解决了。但是我在PartialReID数据集上的性能测试和你们在论文中写到的有一些差距。可以简单介绍一下你们实验的配置吗？比如使用了哪个损失函数，以及主干网络等。
此外，我找不到partial-iLIDS数据集，找到的OccludedREID数据集的文件都是.tif文件，所以可以网盘分享一下你们的数据集吗？

**Comment by lingxiao-he** [comment]
We have update the code and datasets. You can clone the code, good luck!
